geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/ShutdownCommandDUnitTest.java

```diff
@@ -91,8 +91,25 @@ public void setup() throws Exception {
 
   @Test
   public void testShutdownServers() {
-    String command = "shutdown";
+    executeShutdownServers("shutdown");
+  }
+
+  @Test
+  public void testShutdownServersWithSpecifiedOptionValue() {
```

Comment:
It's great that you added some tests it. But I am not sure this small change needs 3 dunit tests. Adding a simple unit test should be sufficient. Try adding this test to the GfshParserParsingTest:


  @Test
  public void testShutdown() throws Exception {
    String command = "shutdown --include-locators";
    GfshParseResult result = parser.parse(command);
    assertThat(result.getParamValueString("include-locators")).isEqualTo("true");
  }

Reply:
I was in two minds while adding tests. There were 2 tests already, testing  _shutdown_ and _shutdown --include-locators=true_. 
I added 2 more to test, _shutdown --include-locators=false_ and _shutdown --include-locators_.

I will get rid of my 2 tests. I was not aware of GfshParserParsingTest. Thanks for your pointer @jinmeiliao.